nixos/modules/services/networking/syncthing.nix

```diff
@@ -119,15 +134,18 @@ let
     builtins.attrNames
     (lib.subtractLists ["folders" "devices"])
     (map (subOption: ''
-      curl -X PUT -d ${lib.escapeShellArg (builtins.toJSON cleanedConfig.${subOption})} \
-        ${cfg.guiAddress}/rest/config/${subOption}
+      curl -X PUT -d ${
+        lib.escapeShellArg (builtins.toJSON cleanedConfig.${subOption})
+      } ${
+        curlAddressArgs "/rest/config/${subOption}"
+      }
```

Comment:
I don't think this style of of having the string interpolation span multiple lines is necessary here. I think it would be better to keep it in the same style as before.